docs/.vuepress/theme/components/Sidebar.vue

```diff
@@ -62,6 +62,64 @@ export default {
     isActive(page) {
       return isActive(this.$route, page.path)
     }
+  },
+
+  mounted() {
```

Comment:
This works but seems pretty gnarly. I wonder if there's a simpler approach? e.g. since we know the sidebar only displays for large screen sizes, can we not just subtract the footer height from the sidebar height to ensure there's no overlap? Perhaps you've considered alternatives - I'm ok with this approach if you think it's necessary.

Reply:
You're right, it is pretty gross. We could possibly do something here with `position: sticky` but I went with this pattern because of the nuances (e.g. it was taking into account the header and footer height anyway, albeit via CSS). I'll explore `sticky` again, I did have a reason to not use it but it's slipped my mind rn. Either way, I'm very very keen to avoid hard-coding heights

Reply:
Ok, went with position sticky. Had to restructure `Layout.vue` a little, and change some global css which I wasn't too keen on doing initially (mainly removing overflow properties). Looks like nothing has changed as a result, apart from having applying display: block to a wrapper on the homepage. Success!